docs/data/data-grid/pivoting/pivoting.md

```diff
@@ -2,16 +2,138 @@
 title: Data Grid - Pivoting
 ---
 
-# Data Grid - Pivoting [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')🚧
+# Data Grid - Pivoting [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
 
-<p class="description">Rearrange columns and values to view data from multiple perspectives.</p>
+<p class="description">Rearrange rows and columns to view data from multiple perspectives.</p>
 
-:::warning
-This feature isn't available yet, but it is planned—you can 👍 upvote [this GitHub issue](https://github.com/mui/mui-x/issues/214) to help us prioritize it.
-Please don't hesitate to leave a comment there to describe your needs, especially if you have a use case we should address or you're facing specific pain points with your current solution.
+The Data Grid Premium's pivoting feature lets users transform the data in their grid by reorganizing rows and columns, creating dynamic cross-tabulations of data.
+This makes it possible to analyze data from different angles and gain insights that would be difficult to see in the default grid view.
+
+:::success
+If you're new to pivoting, check out the [Understanding pivoting](/x/react-data-grid/pivoting-explained/) page to learn what pivoting is and how it works through interactive examples.
 :::
 
-With the pivoting feature, users would be able to turn a column into a row, transforming the column's values into new columns.
+## Quick start
+
+Pivoting is enabled by default and can be accessed through the icon in the toolbar.
+In the demo below, the pivot panel is already open and some pivoting parameters have been set.
+Use the **Pivot** switch at the top of the panel to toggle pivoting off and on.
+You can drag and drop existing columns in the **Rows**, **Columns**, and **Values** dropdown menus to change how the data is pivoted.
+
+{{"demo": "GridPivotingMultipleValues.js", "bg": "inline", "defaultCodeOpen": false}}
```

Comment:
Debatable nitpick. I'd keep the demos with colored markings on the "Understanding pivoting" page since the primary purpose, as I see it, is to help users grasp how pivoting works.

Reply:
This is an unplanned side-effect because one demo is used in 2 places 😅
There should be no color highlights on the main doc page, I'll fix that 👍🏻

Reply:
Fixed https://deploy-preview-9877--material-ui-x.netlify.app/x/react-data-grid/pivoting/#quick-start